bin/BuildPackages.sh

```diff
@@ -39,7 +49,9 @@ if [ -f gapicon.bmp ]
 fi
 
 # We need any subdirectory, to test if $GAPDIR is right
-SUBDIR=`ls -d */ | head -n 1`
+# dangerous to parse output of ls
+#SUBDIR=`ls -d */ | head -n 1`
+SUBDIR=`find -L ./ -maxdepth 1 -mindepth 1 -type d | sed 's|./||' | head -n 1`
```

Comment:
`find` may not be available on all systems, but OK, we can risk that. But in a regex, `.` has a special meaning, so strictly speaking you should escape it in the `sed` invocation.

But in fact, if you replace `./` by `.` in the `find` invocation, you can just drop the call to `sed`...